addOns/ascanrules/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## [61] - 2024-01-24
 ### Changed 
 - Update reference for Server Side Include (Issue 8262) 
+- The Source Code Disclosure - /WEB-INF Folder rule now includes example alert functionality for documentation generation purposes (Issue 6119).
```

Comment:
Needs to be moved.

Reply:
Ugh, thanks. Tackle it in about 30. Thanks for catching it!